src/server/location.rs

```diff
@@ -124,12 +124,25 @@ impl LocationResult {
         headers.get(GOOG_LOC_HEADER).is_some()
     }
 
-    pub fn region(&self) -> String {
-        self.subdivision.clone().unwrap_or_default()
+    pub fn region(&self) -> Option<String> {
+        if self.country() == "US" {
```

Comment:
Wasn't the conclusion here is we'll continue doing what we're doing, always send region regardless of country (and they'll begin caching it outside of US and correct this in their API docs)? cc @ncloudioj

Reply:
Oh, I thought it was the opposite. I'll defer to you and Nan's decision here.

Reply:
ah, OBE.